platform/security/pdp/security-pdp-xacmlrealm/src/main/java/ddf/security/pdp/xacml/realm/XACMLRealm.java

```diff
@@ -59,12 +61,22 @@
 
     private BalanaPdp pdp;
 
+    private final Parser parser;
```

Comment:
Actually, you only need the parameter passed into the constructor to pass along to the `BalanaPdp` ctor.